src/aws-sdk/index.ts

```diff
@@ -0,0 +1,241 @@
+import { aws_iam } from "aws-cdk-lib";
```

Comment:
I'd prefer if we didn't have logic in any of the `index.ts` files and instead only use them to export